src/lib/pages/proposals/components/ProposalsTableLite.tsx

```diff
@@ -25,8 +26,8 @@ export const ProposalsTableLite = () => {
     isFetchingNextPage,
   } = useProposalsLcd(status[0]);
 
-  const { data: proposalData, isLoading: isProposalDataLoading } =
-    useProposalDataLcd(debouncedSearch);
+  const { data: proposalData, isFetching: isProposalDataLoading } =
```

Comment:
Since, we change from `isLoading` to `isFetching`, I think we should update `isProposalDataLoading` to `isProposalDataFetchin` to make the naming convention more related.